bundles/statistics/statsgrid2016/view/IndicatorFormFlyout.js

```diff
@@ -89,16 +89,19 @@ Oskari.clazz.define('Oskari.statistics.statsgrid.view.IndicatorFormFlyout', func
             const datasets = [];
             ind.selectors.forEach(function (sel) {
                 ind.regionsets.forEach(function (regionset) {
-                    sel.allowedValues.forEach(function (value) {
-                        const data = {};
-                        if (typeof value === 'object') {
-                            data[sel.id] = value.id;
-                        } else {
-                            data[sel.id] = value;
-                        }
-                        data.regionset = regionset;
-                        datasets.push(data);
-                    });
+                    // Has data
+                    if (regionset !== 0) {
```

Comment:
What does 0 mean here? I think all the regionset ids are positive integers?

Reply:
Regionset id was 0 if data doesn't exist, if I remember correctly.